config/routes.rb

```diff
@@ -1960,6 +1960,8 @@
         editor
         export_service_dialogs
         show
+        old_dialogs_edit
```

Comment:
Do we need old_dialogs_edit in the routes.rb still? I don't see any http calls to it, can you check if this can be removed from the routes.